.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml

```diff
@@ -161,3 +161,87 @@ steps:
     name: "Cleanupartifactdir"
     displayName: "Cleanup artifact dir"
     condition: always()
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks windows-nodepool-up AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) CLUSTER=${{ parameters.clusterName }}
+        echo "Windows node are successfully added to v4 Overlay Cluster"
+        kubectl cluster-info
+        kubectl get node
+        kubectl get po -owide -A
+    name: "Add_Windows_Node"
+    displayName: "Add windows node on v4 overlay cluster"
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        set -e
+        echo "Deploying windows CNS on v4 overlay cluster"
+        export WINDOWS_CNS_VERSION=$(make cns-version)
+        envsubst '${WINDOWS_CNS_VERSION}' < test/integration/manifests/cns/daemonset-windows.yaml | kubectl apply -f -
+        echo "wait for CNS pods deployment ready"
+        kubectl rollout status ds azure-cns-win -n kube-system --timeout 5m
+        kubectl get po -owide -A
+        kubectl describe ds azure-cns-win -n kube-system | grep Image
+        echo "Upload CNI"
+        dropgz_version=$(make cni-dropgz-test-version)
+        export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS='windows' ARCH='amd64' OS_VERSION='ltsc2022' CNI_DROPGZ_VERSION=${dropgz_version})
```

Comment:
```suggestion
        export DROP_GZ_URL=$(make cni-dropgz-test-image-name-and-tag OS='windows' ARCH='amd64' OS_VERSION='ltsc2022' CNI_DROPGZ_VERSION=${dropgz_version})
```

Reply:
I see most other places have it with a space:
![image](https://github.com/Azure/azure-container-networking/assets/35265851/8b8ec3b7-f474-4c23-b218-18c78055f993)

Reply:
fixed all places